ripd/ripd.c

```diff
@@ -2800,15 +2807,9 @@ DEFUN_NOSH (router_rip,
        "Enable a routing process\n"
        "Routing Information Protocol (RIP)\n")
 {
-	int ret;
-
 	/* If rip is not enabled before. */
 	if (!rip) {
-		ret = rip_create();
-		if (ret < 0) {
-			zlog_info("Can't create RIP");
-			return CMD_WARNING_CONFIG_FAILED;
-		}
+		rip_create();
```

Comment:
`rip_create()` can fail if `rip_create_socket()` fails. We shouldn't ignore this error.

Reply:
Gotcha, I'll fix this